osf/models/analytics.py

```diff
@@ -65,6 +66,11 @@ class PageCounter(BaseModel):
     total = models.PositiveIntegerField(default=0)
     unique = models.PositiveIntegerField(default=0)
 
+    action = models.CharField(max_length=128, null=False, blank=False, default='download', db_index=True)
+    guid = models.ForeignKey(Guid, related_name='pagecounters', null=True, blank=True, db_index=True)
```

Comment:
I am wondering about using `guid` as the field name! Like perhaps it could be more descriptive in its role, like in the `Comment` model the `fk` to `Guid` is named `target` (and the other `root_target`) -- though maybe that's more confusing.  Like `Guid` is really more of a through table to other things that the `PageCounter` is connected to, right? I think I am overthinking this as I can't really think of a better alternative - don't like `target` cause we use that for so many other things... I dunno, what are your thoughts??

Reply:
I see what you're saying! I'm thinking about `resource`.  I used this where generalizing with NPD, methods that needed to work for both nodes and preprints, so I called it `resource`.  I think I got it from the [JSONAPI spec](https://jsonapi.org/format/#fetching-resources)